packages/db/.env.example

```
@@ -0,0 +1,15 @@
+# Copy to `packages/db/.env` and set real values. Do not commit secrets.
+# This file is loaded by Docker Compose (local Postgres) and by Prisma via `src/data-source.ts`.
+
+########################
+# Local Postgres (Docker Compose)
+########################
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=yourpasswordlocalhost
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The POSTGRES_PASSWORD key should go before the POSTGRES_USER key

(UnorderedKey)